test/tests/unit/backend.ts

```diff
@@ -126,3 +130,132 @@ describe("lumens v3", function () {
     });
   });
 });
+
+describe("ledgers", function () {
+  describe("updateCache", function () {
+    it("should store cache with correct data", async function () {
+      // cleanup
+      await redisClient.del(REDIS_LEDGER_KEY);
+      await redisClient.del(REDIS_PAGING_TOKEN_KEY);
+
+      const ledgers: LedgerRecord = [
+        {
+          paging_token: "101",
+          sequence: 10001,
+          successful_transaction_count: 10,
+          failed_transaction_count: 5,
+          operation_count: 50,
+          closed_at: "2022-01-11T01:06:00Z",
+        },
+        {
+          paging_token: "102",
+          sequence: 10002,
+          successful_transaction_count: 20,
+          failed_transaction_count: 10,
+          operation_count: 100,
+          closed_at: "2022-01-12T01:06:00Z",
+        },
+        {
+          paging_token: "103",
+          sequence: 10003,
+          successful_transaction_count: 30,
+          failed_transaction_count: 20,
+          operation_count: 200,
+          closed_at: "2022-01-12T01:07:00Z",
+        },
+      ];
+
+      await updateCache(ledgers, REDIS_LEDGER_KEY, REDIS_PAGING_TOKEN_KEY);
+
+      let cachedLedgers = await redisClient.get(REDIS_LEDGER_KEY);
```

Comment:
We might replace all of these `let`s with `const` in this file.

Reply:
https://github.com/stellar/dashboard/pull/209/commits/3668f052cf5b80466181349192ea2a67ed6e1e36

thx for catching, if I missed any my next PR adding eslint should pick them up